pmg/models/users.py

```diff
@@ -113,6 +117,10 @@ class User(db.Model, UserMixin):
     def __unicode__(self):
         return unicode(self.email)
 
+    @property
+    def confirmed(self):
```

Comment:
Existing methods use naming like `is_authenticated` and `has_expired`, perhaps `is_confirmed` would be a more consistent name? The former two are also methods, rather than properties.